intg-androidtv/apps.py

```diff
@@ -30,7 +30,7 @@
     "ATV PlayFI": {"url": "market://launch?id=com.phorus.playfi.tv"},
     "ATV Now on TV": {"url": "market://launch?id=org.droidtv.nettvrecommender"},
     "ATV Media": {"url": "market://launch?id=org.droidtv.contentexplorer"},
-    "ATV Browser": {"url": "market://launch?id=com.vewd.core.browserui"},
```

Comment:
Please keep this entry

Reply:
oh, that was unintentional, I fixed my commit